src/test/EigenPod.t.sol

```diff
@@ -262,6 +262,9 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
     function testWithdrawBeforeRestaking() public {
```

Comment:
we should test that new pods have hasRestaked set to true

Reply:
added